### PR TITLE
What if we used 2 spaces, though?

### PR DESCRIPTION
**Pull request by zkxs**
Changes everything to two spaces. Removes human-readable whitespace guidelines from CONTRIBUTING.md as they are superseded by editorconfig.

`dotnet format` doesn't run on csproj or sln files so I've formatted them manually.

**THE PRS WILL CONTINUE**  
**UNTIL MORALE IMPROVES**

### CODE REVIEW

**Comment by EIA485** [comment]
personally i prefer 4 spaces, curios why we are changing it.

**Comment by zkxs** [comment]
I personally don't care one way or the other as long as we're consistent. And I'm not really a C# guy so I don't know what the meta is.

**Comment by XDelta** [comment]
I'd agree on 4 spaces

**Comment by EIA485** [comment]
ive opened pr #63 to update the editor config to match the current contributing.md

**Comment by ljoonal** [comment]
Okay so iirc I just quickly looked at the csproj files, saw 2 spaces, and went "oh okay this is what's been used" without properly checking the cs files x'D  the `end_of_line = lf` was intentional, having files which aren't that wasn't.

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -12,19 +12,17 @@ If you want your PR to be approved and merged quickly, please read the following
...
   - MUST use unix-style (LF) line endings. CR or CRLF are not acceptable.
-- *.sln files: use the Visual Studio defaults
-  - indent using tabs
+- *.sln files
+  - indent using two spaces
   - CRLF line endings
-- *.csproj files: use the Visual Studio defaults
+- *.csproj files
   - indent using two spaces
   - CRLF line endings
```
Comment (ljoonal) [inline .github/CONTRIBUTING.md:28]: .... I really want to have one unified style for everything instead of 3 different ones.

**Comment by zkxs** [comment]
*.csproj and *.sln are now formatted using 2 spaces. I've removed human-readable whitespace guidelines from CONTRIBUTING.md as they are superseded by editorconfig.

This brings the PR in line with tab and 4-space PRs, so I've edited the title and description to match.

**Review by ljoonal — APPROVED** [review]
I mean, I can also agree on 2 spaces since it's not as big as 4, and what I've configured my tab size as

**Comment by EIA485** [comment]
#65 seems to be the best option as it allows you to change the visual based on your preference within your editor

**Comment by zkxs** [comment]
#65 is the ultimate winner.

**Pull request closed without merging**